packages/blockchain-link/src/index.ts

```diff
@@ -190,6 +206,15 @@ class BlockchainLink extends EventEmitter {
         });
     }
 
+    /**
+     * Subscribe for live changes in
+     * - blockchain i.e new blocks mined.
+     * - accounts, addresses i.e. new transactions broadcasted or mined.
+     * - fiatRates
+     * - connection to BE (HANDSHAKE, CONNECT, DISCONNECT)
```

Comment:
```suggestion
     * - connection to BE (CONNECT, DISCONNECT)
```
i explained it in previous comment, HANDSHAKE is not emitted to client

Reply:
Good catch! Sorry, that slipped thru when I was moving it to the `index.ts`.Removed in fc6d755e52306d76082660076f8a91bb9b89ab83.